ament_index_cpp/CMakeLists.txt

```diff
@@ -11,7 +11,8 @@ find_package(ament_cmake REQUIRED)
 add_library(${PROJECT_NAME} SHARED
   src/get_resource.cpp
   src/get_resources.cpp
-  src/get_search_paths.cpp)
+  src/get_search_paths.cpp
+  src/has_resource.cpp)
```

Comment:
This change is already on master. Something must have gone wrong during the rebase.

Same below for other changes.